book/website/community-handbook/consistency/consistency-formatting.md

```diff
@@ -0,0 +1,194 @@
+---
+Content written in HTML may not render properly.
+```
+
+When reformatted to Markdown, the table became cleaner and easier to read:
+
+```{figure} ../../figures/html_to_markdown2.png
+---
+name: html_to_markdown2
+alt: A screenshot of a table converted to Markdown from HTML. The table becomes easier to read and understand when converted to Markdown.
+---
+Converting HTML to Markdown makes The Turing Way book easier to read.
+```
+```{note} A PR that addresses this check can be found [here](https://github.com/alan-turing-institute/the-turing-way/pull/1460).
+```
+
+Chapter content written in `HTML` are usually enclosed in tags which begin and end with angle brackets `<>`. 
+[W3Schools](https://www.w3schools.com/html/html_elements.asp) is an excellent resource for understanding what these tags mean, and Markdown reference guides, such as [this cheatsheet](https://www.markdownguide.org/cheat-sheet/), can help translate `HTML` formatting to Markdown.
+There are also helpful [tools](https://jmalarcon.github.io/markdowntables/) that convert `HTML` to Markdown with a single click.
+
+Please note that if `HTML` seems the only option for you to format your text the way you desire, you can only use it if the content in the online book can still be read and understood (use the Netlify preview in your PR to test). 
+For example, [superscripts and subscripts](https://support.squarespace.com/hc/en-us/articles/206543587-Markdown-cheat-sheet#toc-superscript-and-subscript) can be written in `HTML` because they always appear as intended.
+
+#### Writing Checklists
+
+When writing a new chapter for _The Turing Way_, you might include a Checklist subchapter that itemises key action points you want readers to take based on the chapter content.
+For earlier chapters in _The Turing Way_, this subchapter is written in Markdown as a Task List with checkboxes:
+
+```
+# Checklist
+- [ ] Item One
+- [ ] Item Two
+- [ ] Item Three
+
+```
+
+However, the checkboxes do not display as intended in the online version of the book, for example:
+
+```{figure} ../../figures/checklist-formatting.png
+---
+name: checklist-formatting
+alt: When the Checklist subchapter of any chapter is written in Markdown as a Task List with checkboxes, square brackets are displayed rather than checkboxes in the web version of the book.
+---
+Checkboxes are displayed as square brackets in the web version of the book.
+```
+
+Thus, we recommend that you format your Checklist subchapters as unordered lists in your future contributions, and edit earlier chapters that follow the other convention:
+
+```
+# Checklist
+- Item One
+- Item Two
+- Item Three
+
+```
+
+
+(ch-consistency-formatting-hr-headers)=
+### Check 2: Use headers in sequential order.
+
+Non-consecutive headers refer to an increase in header levels of more than one. 
+For example:
+
+```
+# Heading
```

Comment:
Maybe nice to have a table, where the Markdown syntax and rendered text are side-by-side (especially for new contributors). I think now it's just the rendered header? For those unfamiliar with Markdown, the 'increasing' level of headers can be counterintuitive (where the 'higher' header level is actually smaller section headers).

Reply:
True. Although, it might be difficult to show rendered text side by side in that manner using a markdown table. I will explore how we can do this with an image instead